cpp/src/Slice/Parser.h

```diff
@@ -104,7 +104,6 @@ namespace Slice
     bool containedEqual(const ContainedPtr& lhs, const ContainedPtr& rhs);
 
     using TypeList = std::list<TypePtr>;
-    using ExceptionList = std::list<ExceptionPtr>;
```

Comment:
If you expand the few lines below this, you'll see that we double-declare `ExceptionList`.
I guess we wanted to make really sure it got set.